source/docs/software/roborio-info/index.rst

```diff
@@ -6,6 +6,7 @@ roboRIO
 
    roborio-introduction
    /docs/zero-to-robot/step-3/imaging-your-roborio
```

Comment:
We should add a link (note at the top of the article) to the regular imaging page to the roborio2 page for those who get hotlinked.